examples/with-vite-plugin-pwa/astro.config.mjs

```diff
@@ -1,9 +1,79 @@
 import { defineConfig } from 'astro/config';
-import { VitePWA } from 'vite-plugin-pwa';
+import pwa from '@astrojs/pwa';
+import replace from '@rollup/plugin-replace';
```

Comment:
Do we need `@rollup/plugin-replace`? Would just using [`vite.define`](https://vitejs.dev/config/#define) work?

Reply:
I'll take a look later